meta/AMuon/Analytic_Loop_Functions.m

```diff
@@ -22,6 +22,86 @@
 
 *)
 
+(*One Loop Functions*)
+OneLoopA[x_] :=
+ If[x == 0, 2,
```

Comment:
Something to think about. Maybe instead of `if`s we could write full specializations
```mathematica
OneLoopA[0] = 2;
OneLoopA[1] = 3/2;
```